test_runner/random_test_runner/Readme.md

```diff
@@ -0,0 +1,213 @@
+  test_cases:
+    - seed: 1281242544
+    - seed: 3644198185
+    - seed: 2673087374
+    - seed: 1312244741
+    - seed: 2518911638
+```
+
+## How to replay
+
+Prerequisites:
+1. Build as instructed in [How to build](#how-to-build)
+2. Acquire `result.yaml` file:
+   1. Either by running test as stated in [How to run](#how-to-run) part of instruction
+   2. Receiving it from someone who already ran it
+3. Place `result.yaml` in `<some_directory>` IMPORTANT: Do not change filename
+4. Execute:
+ 
+```shell
+ros2 launch random_test_runner random_test.launch.py input_dir:=<some_directory>
+```
+
+Random test runner will load `result.yaml` file and rerun test.
+
+## Running with unity
+
+To run with unity: 
+1. Run unity project that supports `scenario_simulator` integration
+2. Make sure that package name in `map_name` parameter is compatible with used unity project. In case of `kashiwanoha_map` use [Kashiwanoha project](https://gitlab.com/robotec.ai/tieriv/kashiwanoha)
+3. Execute `random_test_runner` launch with `simulator_type` parameter:
+```shell
+ros2 launch random_test_runner random_test.launch.py simulator_type:="unity"
+```
+|  NOTE: Since currently unity integration does not support ego vehicle, `random_test_runner` does not spawn it. |
+|----------------------------------------------------------------------------------------------------------------|
+
+## Launch arguments
+
+This section describes arguments accepted by launch file. Please note that those arguments should not be included in parameters file specified in `test_parameters_filename`.
+
+Apart from that, any `random_test_runner` node parameters can be supplied as a launch arguments.
+Please refer to [Node parameters](#node-parameters) chapter for more details on parameter source priorities.
+
+### General arguments
+
+| Parameter name               | Default value                 | Description                                                                                                                                                                                                                 |
+|------------------------------|-------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `test_parameters_filename`   |  `""`                         | Yaml filename within `random_test_runner/param directory` containing test parameters. With exception from [Launch arguments](#launch-arguments) chapter, parameters specified here will override values passed as arguments |
+| `simulator_type`             |  `"simple_sensor_simulator"`  | Backend simulator. Supported values are `unity` and `simple_sensor_simulator`. It's also accepted by the node but should be supplied as direct launch argument                                                              |
```

Comment:
This part is confusing: `It's also accepted by the node but should be supplied as direct launch argument`

Why does this parameter exist in two places? Is it necessary?